mGAP/src/org/labkey/mgap/mGAPModule.java

```diff
@@ -107,7 +108,8 @@ public void doStartupAfterSpringConfig(ModuleContext moduleContext)
 
         SystemMaintenance.addTask(new mGapMaintenanceTask());
 
-        ContentSecurityPolicyFilter.registerAllowedConnectionSource(this.getClass().getName(), "https://*.fontawesome.com", "https://code.jquery.com", "https://www.gstatic.com");
+        ContentSecurityPolicyFilter.registerAllowedSources(Directive.Connection, this.getClass().getName(), "https://*.fontawesome.com", "https://code.jquery.com");
```

Comment:
Are these needed in `connect-src`? I suspect they're loading JS and font resources, not serving as the endpoints for AJAX-style calls.

Reply:
fair point. it's legacy from the original registerAllowedConnectionSource(). I will prune that.

Reply:
turns out fontawesome.com was actually needed.